src/components/Picklist/pageObject/index.js

```diff
@@ -99,63 +67,39 @@ class PagePicklist {
             .getValue();
     }
 
-    /**
-     * Get the number of registered options.
-     * @method
-     * @returns {number}
-     */
-    getOptionsLength() {
-        return $(this.rootElement).$$('li[data-selected="false"]').length;
-    }
-
-    /**
-     * Returns a new PicklistOption page object of the element in item position.
-     * @method
-     * @param {number} optionIndex - The base 0 index of the PicklistOption.
-     */
-    getOption(optionIndex) {
-        const activeOptions = $(this.rootElement).$$('li[data-selected="false"]');
-        const option = activeOptions[optionIndex];
-        if (option && !option.error) {
-            return new PagePicklistOption(option, this[privateGetMenuBoundsRect]());
-        }
-        return null;
-    }
-
     /**
      * Returns true when the options menu is open, false otherwise.
      * @method
      * @returns {bool}
      */
     isMenuOpen() {
-        return $(this.rootElement)
-            .$('[role="listbox"]')
-            .isDisplayed();
+        return (
+            $(this.rootElement)
+                .$('div[role="combobox"]')
+                .getAttribute('aria-expanded') === 'true'
+        );
     }
 
     /**
-     * Returns the boundaries of Picklist dropdown menu.
+     * Wait until the options menu is open.
      * @method
-     * @returns {object}
      */
-    [privateGetMenuBoundsRect]() {
-        const menu = $(this.rootElement).$('[role="listbox"]');
-        const { x, y } = menu.getLocation();
-        const { width, height } = menu.getSize();
-        return {
-            left: x,
-            top: y,
-            right: x + width,
-            bottom: y + height,
-        };
+    waitUntilOpen() {
+        browser.waitUntil(() => this.isMenuOpen());
     }
 
     /**
-     * Wait until the options menu is open.
+     * Returns a new InternalDropdown page object for the element with the supplied id.
      * @method
      */
-    waitUntilOpen() {
-        browser.waitUntil(() => this.isMenuOpen());
+    getMenu() {
```

Comment:
can we use this method as internal order to avoid calling it when need to get an option?
If it will be used internally then you can use Symbol in order to hide this method